.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "3.6"
+
+install:
+  - pip install pipenv
+  - make install
+
+script:
+  - make lint
```

Comment:
I would run this on `before_script`, what do you think?

Reply:
Fixed 5ca3b05